packages/replayio/src/utils/initialization/checkForNpmUpdate.ts

```diff
@@ -1,20 +1,20 @@
-import { exec } from "child_process";
-import { promisify } from "util";
-import { version as currentVersion, name } from "../../../package.json";
+import { version as currentVersion, name as packageName } from "../../../package.json";
 import { shouldPrompt } from "../prompt/shouldPrompt";
 import { debug } from "./debug";
 import { UpdateCheck } from "./types";
 
-const execAsync = promisify(exec);
-
 const PROMPT_ID = "npm-update";
 
 export async function checkForNpmUpdate(): Promise<UpdateCheck<string>> {
   try {
-    const { stdout: text } = await execAsync(`npm info ${name} --json`, {
-      encoding: "utf8",
+    // https://github.com/npm/registry/blob/master/docs/responses/package-metadata.md#abbreviated-metadata-format
+    const response = await fetch(`https://registry.npmjs.org/${packageName}`, {
+      headers: {
+        Accept: "application/vnd.npm.install-v1+json",
+      },
     });
-    const { version: latestVersion } = JSON.parse(text.trim());
+    const json = (await response.json()) as any;
+    const latestVersion = json["dist-tags"]?.latest;
```

Comment:
I'd say that we should either trust this response or not trust it all. `?.` gives me a vibe that we don't trust it whereas `hasUpdate: currentVersion !== latestVersion` gives me a vibe that we *do* trust it (but in such a case it's inconsistent with the optional chain - and if we think that optional chain is good here then i'd probably add an early return here for `typeof latestVersion !== 'string'`)